### ISSUE TITLE
Subtitles sometime can't be loaded..keep choosing the sub and pressing apply but nothing happens

### ISSUE THREAD

**Issue by SANCTI-afk** [recloudstream/cloudstream#31]
### Steps to reproduce

..

### Expected behavior

 subs should be applied 

### Actual behavior

Subs don't apply 

### Cloudstream version

3.1.3

### Android version

9 and 10

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/recloudstream/cloudstream/releases)**.
- [X] If related to a provider, I have checked the site and it works, but not the app.
- [X] I will fill out all of the requested information in this form.

**Comment by Blatzar** [comment]
A lot of people have had issues loading subtitles because of their networking, the subtitle domain is simply blocked. 

Disable any sort of device wide adblocking solution such as: Adaway or an adblocking DNS (perhaps you set this on the router).
If you are running some sort of VPN try another one and try without VPN.

**Comment by Blatzar** [comment]
On dopebox, sflix.to, solarmovie and HDToday the subtitles from the source does not work because the server simply times out. It does not work on the sites. Always check the site first before reporting here.

**Comment by SANCTI-afk** [comment]
Yes I did set-up my dns for router settings to AdBlocker DNS I don't know if that's included in  the logcat or not but still far from a good damn right guess ..you are my watchdog feature if you permit that likening..I will try to run the process under normal conditions and to update this ticket 
 with my experience

**Comment by SANCTI-afk** [comment]
No.. it's not about DNS sadly ..tested again on
(All movies for you) with several movies and series not even once managed to function...

**Comment by Blatzar** [comment]
Once again not enough information, what movies.

**Comment by Amongog** [comment]
I was watching The Boys las EP from season 1 on Dopebox (emulating from Windows 11 WSA) yesterday.
Today I playe the second season and there are no subtitles loading. Went back to the last ep on season 1, and now it doesn't show up either.

I don't have any kind of adblocking or VPN. I have the latest pre-release https://github.com/recloudstream/cloudstream/commit/f462e1896b6c353baa49a9d8293a241d9008c770.

**Comment by Blatzar** [comment]
> Today I playe the second season and there are no subtitles loading

See: https://github.com/recloudstream/cloudstream/issues/31#issuecomment-1221373140

**Comment by CaptainKnox** [comment]
I was watching The Office on Dopebox. Subtitles loaded for almost half of the episodes in S8. Thereafter, subtitles are not loading and experiencing similar issues as others. Tried with other sources namely VidScr and OpenVids. Same result.

**Comment by SANCTI-afk** [comment]
For a provider like (all movies for you)
There is no subtitles at all ..

> Once again not enough information, what movies?

No subtitles at all regardless how randomly I keep choosing titles to test there is no subs at all

**Comment by SANCTI-afk** [comment]
Hope you could add a plug-in for Subscenes it's more Rich and who knows it could be more reliable and functional

**Comment by Blatzar** [comment]
Look there is a big difference between no subtitles to select from and the selected subtitles not loading. If it's the latter it's simply the case of the source we scrape not having them or they updated their api. If the selected subtitles do not load then the issue might be something bigger.

> subtitles are not loading and experiencing similar issues as others

@CaptainKnox 
As I have said earlier it's an issue with their site, not the app. We cannot fix until it works on their site.

**Comment by SANCTI-afk** [comment]
It depends some providers it doesn't load and others there no subs to select ..I know that's an issue form the other side not yours but the player could easly solve this like mx player it goes to search for subs for you automatically if the name of the media was correct and accurate then you download it and vola you loaded it..so if you could make the player just search for subs with the name of the running media the issue would be solved

**Comment by SANCTI-afk** [comment]
I'm not a native English it's just that hard for me to dismiss that the media I'm watching should be subbed ..moreover Arabic providers don't offer a big database like English providers expect for one called "egybest" which isn't working in the app and the site itself has so many pop-ups

**Comment by SANCTI-afk** [comment]
Could you please put a link for the arabic providers repo I wish I could make a contact with them about " egybest"  provider ..if that one worked I don't want subs anymore as It's the most rich database for subbed content. but the the site itself not affordable so many pop-ups and YT redirects

**Comment by Isxariifshe** [comment]
> For a provider like (all movies for you)
> There is no subtitles at all ..
> 
> > Once again not enough information, what movies?
> 
> No subtitles at all regardless how randomly I keep choosing titles to test there is no subs at all

I think superstream does not fail to load subtitles, use that provider
